tests/Languages/Sql/SqlLanguageTest.php

```diff
@@ -15,7 +16,7 @@ public function test_highlight(string $content, string $expected): void
     {
         $highlighter = new Highlighter();
 
-        $this->assertSame(
+        self::assertSame(
```

Comment:
Due in part to my analysis of the phpunit project, in 2019 [it was decided](https://github.com/sebastianbergmann/phpunit/issues/1914#issuecomment-471978001) that PHPUnit assertions should be rewritten from `$this->assert()` to `self::assert()`.

Reply:
Just FYI, the docs state the following: 

> A common question, especially from developers new to PHPUnit, is whether using $this->assertTrue() or self::assertTrue(), for instance, is “the right way” to invoke an assertion. The short answer is: there is no right way. And there is no wrong way, either. It is a matter of personal preference.

https://docs.phpunit.de/en/11.0/assertions.html#static-vs-non-static-usage-of-assertion-methods

Reply:
Yeah, it's one of those things where old-school, pro-dynamic-preprocesser folks who hate typehinting and property types really prefer the $this-> while those who see the value of static analysis prefer self::.

Since the assert() methods of PHPUnit were separated into the PHPUnit\Assert standalone library since PHPUnit 7, if one prefers the more strict nature of modern PHP, static calls are the way to go.

PHPUnit is officially neutral on the heated topic.

Reply:
@hopeseekr just a general remark: I've noticed you being rather pushy in many issues about your own opinion. Everyone can have their own point of view, but I'm really not in a place where I can afford to make every issue and decision into a long discussion. I have other things going on besides this project, and want to spend my time on it as productive as possible.

Hope you understand, and I hope you still find joy in contributing here, I really appreciate the pr's you've sent up until this point!